requirements.txt

```diff
@@ -1,13 +1,10 @@
+git+https://github.com/EGA-archive/crypt4gh.git
 pika==1.0.1
 aiohttp==3.3.2
```

Comment:
also aiohttp, looks a few version old ...

Reply:
in setup there is `aiohttp>=3.3.2` requirements seems out of sync with `setup.py`

Reply:
That's correct, I have not changed that. It's not the purpose of that PR.